packages/sdk/src/types/Events.ts

```diff
@@ -0,0 +1,23 @@
+export interface EventsParams {
+	event?: string
+	customer?: {
+		source_id?: string
+		id?: string
+		name?: string
+		email?: string
+		metadata?: Record<string, any>
+	}
+	metadata?: Record<string, any>
+	referral?: {
+		code?: string
```

Comment:
If you already defined referral then it would be wise to define `code` since it does not really make sense to use `referral: {}`